routes/api/user/book.js

```diff
@@ -5,83 +5,83 @@ const User = require('common/User')
 const Book = require('common/Book')()
 
 module.exports = {
-	get: function(req, res){
-		if(!req.session.passport.user){
-			return res.status(500).send()
-		}
-		let user = User(req.session.passport.user)
-		let page = Number(req.query.page)
+  get: function(req, res){
+    if(!req.session.passport.user){
+      return res.status(500).send()
+    }
+    let user = User(req.session.passport.user)
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  Missing 'new' prefix when invoking a constructor.

*  Missing semicolon.